packages/styles/alert.css

```diff
@@ -1,22 +1,29 @@
 :root {
-  --alert-warning-background-color: #d93251;
-  --alert-warning-header-text-color: #fff;
+  --alert-default-color: #3c7aae;
+  --alert-warning-color: #d93251;
+
+  --alert-default-color-dark: #6cdaf2;
+  --alert-warning-color-dark: #fe6d6b;
```

Comment:
This doesn't follow our theming guidelines, see: https://cauldron.dequelabs.com/style-guidelines#theming

The correct way to do this is to style the "default" values on `:root` and have an additional selector to match the more specific styles. Something like:

```css
:root {
  --alert-border-color: ...
  --alert-warning-border-color: ...
}

.cauldron-theme-dark {
  --alert-border-color: ...
  --alert-warning-border-color: ...
}
```

Additionally, these should be pulling from global [`variables.css`](https://github.com/dequelabs/cauldron/blob/develop/packages/styles/variables.css) colors, example below:

```css
--alert-border-color: var(--accent-primary)
```

Additionally, removal of existing css variables would constitute a breaking change per our [contributing guidelines](https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md#breaking-changes). We can use new variables that better fit the new component, but need to keep the existing ones around to accommodate a deprecation period.

Reply:
Thank you for highlighting this, I've added previous colors back and updated colors according to your suggestions. I've separated the colors for borders and icons, even though they are identical. Not sure if that's the right way to go, but I thought it might help if these colors change in the future in any way. If that's not acceptable, I'll change it back to use one variable.

Reply:
Also, I needed to make a couple of changes to dialog styles, where I saw these theming guidelines weren't followed fully :) It was preventing my changes from taking effect in dark mode.

Reply:
> Not sure if that's the right way to go, but I thought it might help if these colors change in the future in any way. If that's not acceptable, I'll change it back to use one variable.

I could go either way, but what you have is fine as it provides more granularity.